packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -33,16 +32,31 @@ export default class SquirrelWindowsTarget extends Target {
 
     const files = await fs.promises.readdir(tmpVendorDirectory)
     for (const file of files) {
-      if (["Squirrel.exe", "StubExecutable.exe"].includes(file)) {
+      if (file === "Squirrel.exe") {
         const filePath = path.join(tmpVendorDirectory, file)
         log.debug({ file: filePath }, "signing vendor executable")
         await this.packager.sign(filePath)
+        break
       }
     }
-
     return tmpVendorDirectory
   }
 
+  private async generateStubExecutableExe(appOutDir: string, vendorDir: string) {
+    const files = await fs.promises.readdir(appOutDir, { withFileTypes: true })
+    for (const file of files) {
+      if (path.extname(file.name) === ".exe" && path.basename(file.name, "exe") !== "Squirrel") {
+        const filePath = path.join(appOutDir, file.name)
+        log.debug({ file: filePath }, "generating stub executable for exe")
+        const fileNameWithoutExt = file.name.slice(0, -4)
```

Comment:
Can extract `path.basename(file.name, "exe")` from if-statement to a `const` and then reuse here instead of using `slice` since `basename` w/ ext supplied **should** get us the `filenameWithoutExt`